js/ui/form/instance_storage.js

```diff
@@ -0,0 +1,65 @@
+import Guid from "../../core/guid";
+import { each } from "../../core/utils/iterator";
+
+class InstanceStorage {
+    constructor() {
+        this._storage = {};
+    }
+
+    _find(condition) {
+        var resultInstance;
+
+        each(this._storage, function(guid, { instance, item }) {
+            if(condition(item)) {
+                resultInstance = instance;
+
+                return false;
+            }
+        });
+
+        return resultInstance;
+    }
+
+    clear() {
```

Comment:
I suggest to lift public interface upper

Reply:
I've looked to another classes, for example:
https://github.com/DevExpress/DevExtreme/blob/18_2/js/ui/accordion.js#L486
https://github.com/DevExpress/DevExtreme/blob/18_2/js/ui/autocomplete.js#L256
private methods go first